starters/plugin/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "gatsby-plugin-starter",
```

Comment:
This naming is probably going to be a problem, because the `gatsby-plugin-*` pattern is used for full-on plugins.

I don't have a strong opinion about this, but maybe `gatsby-template-plugin`?